vendor/commento.js

```diff
@@ -243,7 +243,7 @@ init_commento = function(server) {
     COMMENTO_SERVER = server;
     loadJS("https://cdn.rawgit.com/showdownjs/showdown/1.6.3/dist/showdown.min.js", function() {
         loadCSS("https://cdn.rawgit.com/picturepan2/spectre/master/docs/dist/spectre.min.css");
-        loadCSS("https://cdn.rawgit.com/adtac/commento/0.0.2/vendor/commento.min.css");
+        loadCSS(server + "/assets/commento.min.css");
```

Comment:
`vendor/commento.min.css`, no? Or should we rename the `vendor` directory to `assets`?